shared/chat/conversation/input-area/normal/wallets-icon/container.js

```diff
@@ -29,8 +29,13 @@ const mapDispatchToProps = dispatch => ({
 
 const renderNothingProps = {shouldRender: false}
 const mergeProps = (stateProps, dispatchProps, ownProps) => {
-  if (stateProps._meta.teamType !== 'adhoc' || stateProps._meta.participants.size !== 2) {
-    // Only show this for one-on-one conversations
+  // Only show this for pending (size == 1) or one-on-one (size == 2)
+  // conversations.
+  if (
+    stateProps._meta.teamType !== 'adhoc' ||
+    stateProps._meta.participants.size < 1 ||
```

Comment:
Not entirely sure why there's this distinction.

Reply:
Why is the participants of size 1 for the resolved pending conversation?

Reply:
I think it's just how we build the list of chat participants for a new chat. @chrisnojima any particular reason? Come to think of it, I don't think this code is quite right -- it should count the number of participants minus self, and show the $ when that number is 1.